src/storage/txn/commands/mod.rs

```diff
@@ -404,13 +404,18 @@ fn find_mvcc_infos_by_key<S: Snapshot>(
 ) -> Result<LockWritesVals> {
     let mut writes = vec![];
     let mut values = vec![];
+    debug!("finding lock");
```

Comment:
How about remove all new added debug log? Seems they are only for developer's debug.